oe_corporate_site_info.module

```diff
@@ -6,3 +6,95 @@
  */
 
 declare(strict_types = 1);
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\rdf_skos\Entity\ConceptInterface;
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function oe_corporate_site_info_form_system_site_information_settings_alter(&$form, FormStateInterface $form_state, $form_id): void {
+  $corporate_site_info = \Drupal::config('oe_corporate_site_info.settings');
+  $skos_concept_storage = \Drupal::entityTypeManager()->getStorage('skos_concept');
+
+  $form['oe_site_information'] = [
+    '#type' => 'details',
+    '#title' => t('Corporate site information'),
+    '#open' => TRUE,
+  ];
+
+  $site_owner = $corporate_site_info->get('site_owner') ?? NULL;
+  $site_owner_entity = NULL;
+  if ($site_owner) {
+    $site_owner_entity = $skos_concept_storage->load($site_owner);
+  }
+
+  $form['oe_site_information']['site_owner'] = [
+    '#title' => t('Site owner'),
+    '#type' => 'entity_autocomplete',
+    '#target_type' => 'skos_concept',
+    '#selection_handler' => 'default:skos_concept',
+    '#selection_settings' => [
+      'concept_schemes' => [
+        'http://publications.europa.eu/resource/authority/corporate-body',
+      ],
+      'match_operator' => 'CONTAINS',
+      'match_limit' => 10,
+    ],
+    '#validate_reference' => FALSE,
```

Comment:
Is this #validate_reference part of your element type? If not, it doesnt belong here. Check all the rest and only add what makes sense please.

Reply:
Removed unused elements. Looks like it is the last leftover. Did not find other useless array elements.